.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  web:
+    volumes:
+      - .:/app:delegated
+      - vscode-server:/root/.vscode-server:cached
+      - /var/run/docker.sock:/var/run/docker.sock:cached
+    # Override command to prevent container crashing if webpack build exits
+    command: sleep infinity
```

Comment:
Note that this overrides the CMD in the Dockerfile, so the user will need to run start_server.sh manually. If you want to leave this out and use the standard CMD, that's fine with me.